test/gpu/native/studies/minibude/SKIPIF

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Skip if not doing performance testing
+if [ -z "$CHPL_TEST_PERF" ]; then
+  echo "True"
+  exit
+fi
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+MINIBUDE_URL=${MINIBUDE_URL:-git@github.com:xianghao-wang/miniBUDE.git}
+MINIBUDE_BRANCH=${MINIBUDE_BRANCH:-benchmark}
```

Comment:
@milthorpe mentioned [here](https://github.com/chapel-lang/chapel/pull/24073#issuecomment-1960671222) that the current version is in a different repo. Could you check whether the patch and the setup here works for that before merging?

Reply:
Good point. I've updated things to use <https://github.com/milthorpe/miniBUDE/tree/v2/src/chapel>.

The code is a bit different so I've had to modify the patch file somewhat.